scanner/families/infofinder/types/info.go

```diff
@@ -30,12 +30,7 @@ type Info struct {
 	Data string   `json:"data"`
 }
 
-type ScannerResult struct {
-	ScannerName string
-	Infos       []Info
-	Error       error
-}
-
-func (sr ScannerResult) GetError() error {
-	return sr.Error
+type FlattenedInfo struct {
```

Comment:
why `FlattenedInfo ` here? at some point we want to have the `scannerName ` in a parent object, right? maybe just use `Info`